tests/test_make.py

```diff
@@ -302,6 +302,22 @@ class D(object):
         assert C.D.__name__ == "D"
         assert C.D.__qualname__ == C.__qualname__ + ".D"
 
+    def test_post_init(self):
```

Comment:
`make_class` already supports it.  A `__post_init__` key can be defined with the dictionary passed in to make_class:

```python
>>> import attr
>>> def foo(self):
...     print('in foo')
...
>>> C1 = attr.make_class('C1', {'x': attr.ib(default=42), '__post_init__': foo})
>>> c = C1()
in foo
```